src/uikit.cs

```diff
@@ -5379,6 +5832,12 @@ interface UIFont : NSCopying {
 		[Internal] // bug 25511
 		IntPtr _FromName (string name, nfloat size);
 
+		[iOS (13,0), TV (13,0)]
+		[Static]
+		[Internal] // bug 25511
```

Comment:
Just c/p the one above but I will look for it. IIRC it was the one where the life of the UIFont mattered because apple was reusing it under the hood, something around those lines.

Reply:
OK, that made me look here: https://bugzilla.xamarin.com/show_bug.cgi?id=25511, which is the right one. I still think the full url should be in the comment :smile:

Reply:
I did change it, it just did not reflect here

```
			[iOS (13,0), TV (13,0), Watch (6,0)]
			[Static]
			[Internal] // bug https://xamarin.github.io/bugzilla-archives/25/25511/bug.html
			[Export ("monospacedSystemFontOfSize:weight:")]
			IntPtr _MonospacedSystemFontOfSize (nfloat fontSize, double weight);
```